### PR TITLE
Make virtual disk size configurable via the CLI

### PR DESCRIPTION
**Pull request by nickryand**
I have been playing around with omicron in a home lab and ran into an issue with the virtual disks being to small. This lets the size to be overridden via the CLI.

### CODE REVIEW

File: dev-tools/xtask/src/virtual_hardware.rs [hunk 1]
```diff
@@ -90,13 +90,16 @@ pub struct Args {
     #[clap(long, default_value = "/var/tmp")]
     vdev_dir: Utf8PathBuf,
 
+    /// Allow customization of the vdev size
+    #[clap(long, default_value_t = 20 * GB)]
+    vdev_size: u64,
+
```
Comment (iliana) [inline dev-tools/xtask/src/virtual_hardware.rs:96]: 1. This argument is only used in (and only makes sense in) the context of creating virtual hardware, so it should be part of the `Commands::Create` variant.
2. I would reword the doc comment as it's used to create the `--help` output. Perhaps something like `Size in bytes for created vdevs`.

**Review by iliana — COMMENTED** [review]
Thanks for the suggestion! If you'd like to make the change suggested (and also [double-check `cargo fmt`](https://github.com/oxidecomputer/omicron/actions/runs/10582645458/job/29379000528?pr=6450)) below I'd be happy to integrate this. I can also commit the change to your branch and merge it, whichever you prefer.

File: dev-tools/xtask/src/virtual_hardware.rs (hunk 1 above)
Comment (nickryand) [reply dev-tools/xtask/src/virtual_hardware.rs]: I have pushed the fmt update and moved the argument to the proper variant. No need for you to take time to do that. Thanks for the feedback!

**Comment by iliana** [comment]
`cargo fmt` once more? 😅 That last commit dropped enough characters to make that fit the 80-char width.

**Comment by nickryand** [comment]
hah sorry done. I'm noobing it up apparently.

**Review by iliana — APPROVED** [review]
